pyproject.toml

```diff
@@ -24,8 +29,8 @@ i = [
   { cmd = "pre-commit install" }
 ]
 start = "uvicorn --app-dir=src main:app --reload"
-format = "black src"
-format_check = "black --check src"
+format = "black ."
+format_check = "black --check ."
```

Comment:
Smaller PR's next time, or at least open this one up as your working so 29 files arnt dumped all at once.

Reply:
> Smaller PR's next time, or at least open this one up as your working so 29 files arnt dumped all at once.

It's only this big because they're all under the same task and this is just setting up the initial models

Reply:
Its big because you purposefully didn't share the progress until it was already done

Reply:
This is all just basic ORM models setup. I didn't share the progress because it's something basic that took me less than 5 hours to complete and the only reason I didn't make a PR when I was done is because it's good practice to test it first and make sure it's working, which was what I did before making the PR, as well adding in the changes we discussed on Tuesday. I also gave an update in the discord channel.